Runtime/Scripts/GltfImport.cs

```diff
@@ -1260,37 +1284,45 @@ public class GltfImport : IGltfReadable, IGltfBuffers {
             }
 
             if(images!=null && gltfRoot.textures!=null) {
+                SamplerKey defaultKey = new SamplerKey(new Sampler());
                 textures = new Texture2D[gltfRoot.textures.Length];
-                var imageVariants = new Dictionary<int,Texture2D>[images.Length];
-                for (int textureIndex = 0; textureIndex < gltfRoot.textures.Length; textureIndex++)
+                var imageVariants = new Dictionary<SamplerKey,Texture2D>[images.Length];
+                for (int textureIndex = 0; textureIndex < gltfRoot.textures.Length; textureIndex++) 
```

Comment:
I write it here once, but there are more occasions below:
I won't accept white space changes. They make diffs and annotations hard to read and distract from actual change.

Reply:
It should be okay now